### PR TITLE
Add new astro cli commands to support RBAC in Orbit

### PR DESCRIPTION
**Pull request by andriisoldatenko**
Fix https://github.com/astronomer/astro-cli/issues/212

### CODE REVIEW

File: cmd/validation.go [hunk 1]
```diff
@@ -67,3 +67,14 @@ func coalesceWorkspace() (string, error) {
...
+
+func validateRole(role string) error {
+	validRoles := []string{"WORKSPACE_ADMIN", "WORKSPACE_EDITOR", "WORKSPACE_VIEWER"}
+
+	for _, validRole := range validRoles {
+		if role == validRole {
+			return nil
+		}
+	}
+	return errors.Errorf("please use one of: %s", strings.Join(validRoles, ", "))
```
Comment (andriisoldatenko) [inline cmd/validation.go]: @schnie Please take a look on error message? sounds good?

File: cmd/workspace.go [hunk 2]
```diff
@@ -80,6 +80,14 @@ var (
 		RunE:  workspaceUserAdd,
 	}
 
+	workspaceUserUpdateCmd = &cobra.Command{
+		Use:   "update user role",
```
Comment (andriisoldatenko) [inline cmd/workspace.go]: @schnie ^

File: cmd/workspace.go [hunk 3]
```diff
@@ -166,10 +189,29 @@ func workspaceUserAdd(cmd *cobra.Command, args []string) error {
...
 	// Silence Usage as we have now validated command input
 	cmd.SilenceUsage = true
+	return workspace.Add(ws, args[0], role)
+}
+
+func workspaceUserUpdate(cmd *cobra.Command, args []string) error {
+	ws, err := coalesceWorkspace()
+	if err != nil {
+		return errors.Wrap(err, "failed to find a valid workspace")
+		// fmt.Println("Default workspace id not set, set default workspace id or pass a workspace in via the --workspace-id flag")
```
Comment (schnie) [inline cmd/workspace.go]: should we delete this commented out line?

File: cmd/validation.go (hunk 1 above)
Comment (schnie) [reply cmd/validation.go]: looks fine to me

File: cmd/workspace.go (hunk 2 above)
Comment (schnie) [reply cmd/workspace.go]: looks good

File: cmd/workspace.go [hunk 4]
```diff
@@ -80,6 +80,14 @@ var (
 		RunE:  workspaceUserAdd,
 	}
 
+	workspaceUserUpdateCmd = &cobra.Command{
+		Use:   "update user role",
+		Short: "Update a user to a workspace",
+		Long:  "Update a user to a workspace",
```
Comment (schnie) [inline cmd/workspace.go]: Maybe "Update a user's role for a workspace"

File: cmd/workspace.go [hunk 5]
```diff
@@ -88,6 +96,13 @@ var (
 		Args:    cobra.ExactArgs(1),
 		RunE:    workspaceUserRm,
 	}
+
+	workspaceUserListCmd = &cobra.Command{
+		Use:   "list",
+		Short: "Print list of user roles",
+		Long:  "Print list of user roles",
```
Comment (schnie) [inline cmd/workspace.go]: How about "List users in a workspace"?

**Comment by andriisoldatenko** [comment]
```
go run main.go workspace user -h
Users can be added or removed from workspaces

Usage:
  astro workspace user [command]

Available Commands:
  add         Add a user to a workspace
  list        List users in a workspace
  remove      Remove a user from a workspace
  update      Update a user's role for a workspace

Flags:
  -h, --help   help for user
```

File: cmd/workspace.go (hunk 4 above)
Comment (andriisoldatenko) [reply cmd/workspace.go]: fixed.

File: cmd/workspace.go (hunk 5 above)
Comment (andriisoldatenko) [reply cmd/workspace.go]: fixed.

File: cmd/workspace.go (hunk 3 above)
Comment (andriisoldatenko) [reply cmd/workspace.go]: done.

**Pull request merged**